puppet/modules/chassis/manifests/wp.pp

```diff
@@ -83,4 +84,9 @@
 	file { '/home/vagrant/.wp-cli/config.yml':
 		content => template('chassis/wp-cli.yml.erb')
 	}
+
+	file { '/vagrant/wp-cli.yml':
+		content => template('chassis/wp-cli.local.yml.erb')
```

Comment:
`wp-cli.local.yml` is a real thing (for overrides), so imo let's call this `wp-cli.yml.erb`, and rename the old one to `wp-cli-config.yml.erb`

Reply:
Awesome. Thanks for that. I originally wanted to change the old file name and go with this naming convention but I wasn't sure what to do! Taa muchly!